### ISSUE TITLE
How to train multiple steps on local machine???

### ISSUE THREAD

**Issue by cauivy** [tensorflow/minigo#532]
Hello, guys!

I am trying to train Minigo on my local machine. I found when I changed the parameter of 'steps_to_train', it seems that the 'train.py' will always run around 20 iterations/steps. Am I right? How can I train multiple steps, e.g., 1000, 10000 etc. over a set of training data?

Thanks in advance!

**Comment by sethtroisi** [comment]
I'd guess that your running out of training examples.
Are you using one of our golden chunks (e.g. [minigo-pub/v9-19x19/data/golden_chunks/](https://console.cloud.google.com/storage/browser/minigo-pub/v9-19x19/data/golden_chunks/)) or something you had produced?

**Comment by cauivy** [comment]
Thanks for your reply!

I just followed the README instructions from 'Training Minigo': 1)bootstrap; 2)selfplay; 3)train. The data for my training was from the selfplay (generated by the bootstrap/random model). 

I double checked the training results and found the 'train.py' just runs 1 iteration/step with the data, no matter what the number of the 'steps_to_train' was set.  Then I tried to loop 'estimator.train(_input_fn, steps=steps, hooks=hooks)' N times in the 'train.py' , and found  it can train N times by this way.  

How can I see how many training examples were generated in each '.tfrecord.zz' file? From my understanding, after running out of a set of training examples (one epoch), it will continue looping the next epoch over the training examples, which should not be ran out of. 

Which '.tfrecord.zz' file I should to use so that I can run multiple training steps by setting 'steps_to_train' or other parameters?

Thanks again!
